infra/main.bicepparam

```diff
@@ -11,4 +11,20 @@ param useKeyVault = bool(readEnvironmentVariable('USE_KEY_VAULT', 'true'))
 
 param authType = readEnvironmentVariable('AZURE_AUTH_TYPE', 'keys')
 
-param hostingModel = readEnvironmentVariable('AZURE_APP_SERVICE_HOSTING_MODEL', 'container')
+param hostingModel = readEnvironmentVariable('AZURE_APP_SERVICE_HOSTING_MODEL', 'code')
+
+// The following are being renamed to align with the new naming convention
+// we manipulate existing resources here to maintain backwards compatibility
+
+// We need the resourceToken to be unique for each deployment (copied from the main.bicep)
+var subscriptionId = readEnvironmentVariable('AZURE_SUBSCRIPTION_ID', 'subscription_id')
+param resourceToken = toLower(uniqueString(subscriptionId, environmentName, location))
+
+// Retrieve the Search Name from the Search Endpoint (if it exists)
+var azureAISearcEndpoint = readEnvironmentVariable('AZURE_SEARCH_SERVICE', 'https://./')
+var searchSerivceName = split(substring(azureAISearcEndpoint, 8, length(azureAISearcEndpoint) - 9), '.')[0]
```

Comment:
Is `length(azureAISearcEndpoint) - 9` the number of characters we want to extract? If so, I think we can leave that value off and it will default to the end of the string

It's a bit hard to understand what it's doing